integration/test/chartvalues/chartvalues_test.go

```diff
@@ -0,0 +1,44 @@
+// +build k8srequired
+
+package chartvalues
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/giantswarm/micrologger"
+
+	"github.com/giantswarm/chart-operator/integration/chart"
+	"github.com/giantswarm/chart-operator/integration/templates"
+)
+
+func TestChartValues(t *testing.T) {
```

Comment:
This is going to be a test to check the chart values, currently just the minimum to make the test pass.

Reply:
So ATM it really doesn't test anything correct?

Reply:
Yes, I want to get the structure down first.